pallets/swap/src/pallet/mod.rs

```diff
@@ -543,7 +546,7 @@ mod pallet {
                         netuid,
                         position_id,
                         liquidity: liquidity_delta,
-                        tao: (result.tao as i64).neg(),
+                        tao: (result.tao.to_u64() as i64).neg(),
```

Comment:
I don't think result.tao.to_u64() as i64 is safe, it will wrap for large numbers